### PR TITLE
Truncate memote test output

### PR DESCRIPTION
**Pull request by stelmo**

### CODE REVIEW

File: src/memote/frontend.jl [hunk 1]
```diff
@@ -13,307 +36,303 @@ run_tests(filename::String, optimizer; kwargs...) = run_tests(
...
             end
-            @testset "Has genes" begin
-                @test Basic.model_has_genes(model)
+            @testset "Has SBO annotated transport reactions" begin
+                @atest any(is_transport_reaction(model, rid) for rid in reactions(model)) "No reactions are SBO annotated as transport reactions."
             end
-            @testset "Has compartments" begin
-                @test Basic.model_has_compartments(model)
+            @testset "Has SBO annotated biomass reactions" begin
+                @atest any(is_biomass_reaction(model, rid) for rid in reactions(model)) "No reactions are SBO  annotated as biomass reactions."
```
[the remark this answers is not in the document]
Comment (exaexa) [reply src/memote/frontend.jl:81]: (also assertions)

**Review by exaexa — CHANGES_REQUESTED** [review]
Small stylistic things. Mostly the assertions should print testable assertions, not error reports. (It still gets reported as an "Expression"!)

File: src/memote/frontend.jl [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+process.
+"""
+function run_tests(
+    model::MetabolicModel,
+    optimizer;
+    config = Config.memote_config,
+    filename = nothing,
+    workers = [myid()],
+)
+    @info "Starting MEMOTE tests..."
```
[the remark this answers is not in the document]
Comment (stelmo) [reply src/memote/frontend.jl:18]: yeah unnnecessary but I let's the user know that something is about to start :)

File: README.md [hunk 3]
```diff
@@ -77,25 +77,27 @@ FBCModelTests.Memote.run_tests("e_coli_core.json", GLPK.Optimizer; workers=worke
...
+details about where/why your model failed certain tests, it is best to capture
+the output, and save it to a file. A convenient way to do this is with
+[ansi2html](https://github.com/agnoster/ansi2html). Additionally, to make the
+output more display friendly, we recommend `run_tests_toplevel` is used instead
+of `run_tests`.
 
 An example workflow entails using the scripts located in `bin/`:
 ```
-fbcmt-memote-run --color=yes -s GLPK -w 6 e_coli_core.xml > e_coli_core.test.out
+julia --color=yes --project=. fbcmt-memote-run -s GLPK -w 6 e_coli_core.xml > e_coli_core.test.out
```
Comment (exaexa) [inline README.md:88]: this expects that people have the project somewhere; suggest removing it

File: bin/fbcmt-memote-run [hunk 4]
```diff
@@ -29,12 +29,7 @@ if help
     exit(1)
 end
 
-if length(freeargs) == 1
-    global model_file = first(freeargs)
-else
-    @error "Expected precisely 1 non-keyword argument: the model location."
-    exit(1)
-end
+global model_file = last(freeargs)
```
Comment (exaexa) [inline bin/fbcmt-memote-run:32]: this is probably the reaction to the problems with env, it's better not to ignore non-last feeargs (or at least print a warning).


TLDR; rollback to error throwing

File: test/memote.jl [hunk 5]
```diff
@@ -15,20 +15,15 @@ using FBCModelTests.Memote.Metabolites
...
-    @test result.errs == 0
-
-    # @testset "Report can be written successfully as JSON" begin
-    #     r = Memote.generate_report(model_file["e_coli_core.json"], GLPK.Optimizer)
-    #     @test r isa Dict
-
-    #     # this should work without errors
-    #     @test JSON.json(r) isa String
-    # end
+    @test result.errs == 1 # broken counts here I think...
```
Comment (exaexa) [inline test/memote.jl:26]: can we document what is the error?

Comment (exaexa) [reply test/memote.jl:26]: (and why)

**Review by exaexa — COMMENTED** [review]
a few minor things

File: README.md (hunk 3 above)
Comment (stelmo) [reply README.md:88]: it means they need an environment

File: test/memote.jl (hunk 5 above)
Comment (stelmo) [reply test/memote.jl:26]: yep, it is the energy generating test, I still need to find a way to make it non-fragile

File: bin/fbcmt-memote-run (hunk 4 above)
Comment (stelmo) [reply bin/fbcmt-memote-run:32]: done

File: README.md (hunk 3 above)
Comment (stelmo) [reply README.md:88]: I dunno, this makes things cleaner at the end of day :/  people need to not be clueless

**Review by exaexa — APPROVED** [review]
lovely
